### ISSUE TITLE
Webpack's require with expression throws

### ISSUE THREAD

**Issue by Izhaki** [zeit/next.js#13203]
# Bug report

## Describe the bug

When using Webpack's [require with expression](https://webpack.js.org/guides/dependency-management/#require-with-expression) like so:

```js
require("./" + name + ".js");
```

It throws with:

```
ReferenceError: name is not defined
```

## To Reproduce

```
$ yarn create next-app
```

`/pages/index.js`:
```js
require("./" + name + ".js");
```

```
yarn build
```

## Expected behavior

This should work.

## System information

- Version of Next.js: 9.4.2 (9.3.6 also fails)
- Version of Node.js: 10.16.0

## Additional info

Note that if you run:

```
$ npx webpack pages/index.js
```

It builds alright.

Also note that `require.context` works fine in next.js.

**Comment by Izhaki** [comment]
Hmmm... This fails with webpack only after running `main.js`.

So not a next issue.

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
